config/ecs/ecs-default-config.yaml

```diff
@@ -1,6 +1,5 @@
 extensions:
   health_check:
```

Comment:
Do we need to specify endpoint and path in this config like the others?

Reply:
I have removed specific mention of endpoint and path from all configs- as pointed out by in one of the comments by Anthony. The default healthcheck endpoints will be applicable in this case.